aten/src/ATen/native/cpu/FunctionOfAMatrixUtilsKernel.cpp

```diff
@@ -0,0 +1,54 @@
+#include <ATen/native/FunctionOfAMatrixUtils.h>
+
+#include <ATen/native/cpu/Loops.h>
+
+#if (defined(_WIN32) || defined(_WIN64))
+#define RESTRICT __restrict
+#else
+#define RESTRICT __restrict__
+#endif
+
+namespace at { namespace native {
+
+namespace {
+
+void _compute_linear_combination_cpu_kernel(
+  TensorIterator& iter,
+  int32_t in_stride,
+  int32_t coeff_stride,
+  int32_t num_summations
+) {
```

Comment:
@ezyang , @ngimel , could you please tell whether `int32_t` is fine in here?

Reply:
It usually isn't (anything 4G or bigger tends to trigger these conditions). And on CPU, there isn't really any cost to making these 64-bit, so you should probably do so.

Reply:
Fixed.